src/lib/src/component/tree-viewer/tree-viewer.component.css

```diff
@@ -32,4 +32,16 @@
 
 .tree-view-item-key:hover {
     background-color: #2A2D2E;
+}
+
+.active {
+    background-color: #3f3f46 !important;
```

Comment:
Yes, thanks we can exclude it with `.tree-view-item-key:not(.active):hover` for the hover styling.